frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts

```diff
@@ -31,7 +31,6 @@ import {StateService} from '@uirouter/core';
 import {Component, Injector, OnInit} from '@angular/core';
 import {WorkPackageViewSelectionService} from 'core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-selection.service';
 import {WorkPackageSingleViewBase} from "core-app/modules/work_packages/routing/wp-view-base/work-package-single-view.base";
-import {BackRoutingService} from "core-app/modules/common/back-routing/back-routing.service";
 import {of} from "rxjs";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/15f41fef25cd2ce7e61e5e731c6e8da20efb2f58/frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>